run/filesystem/gcsfuse_run.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -eo pipefail
+
+# Create mount directory for service
+mkdir -p $MNT_DIR
+
+echo "Mounting GCS Fuse."
+gcsfuse --debug_gcs --debug_fuse $BUCKET $MNT_DIR
```

Comment:
Unless something changed, this won't produce debug output because of some expectations around running in the foreground. Also, this produces *a lot* of debug output so I'd add a comment about removing the --debug_fuse flag at least to reduce verbosity.

Reply:
In my exploration, these flags gave enough logging output to debug.